README.md

```diff
@@ -4,99 +4,91 @@
 
 ### Question #1
 
-Describe the differences between a SQL and NoSQL DB, and when you might use each.
+ Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
-```text
-Your answer...
 
-```
+SQL databases are primarily relational, table-based databases, whereas NoSQL databases are non-relational and primarily use key value pairs and groups. SQL databases have predefined schema for their relationships (as in rails), whereas NoSQL do not have separate pre-defined schema files.    
+ ```
 
-### Question #2
+ ### Question #2
+ What's wrong with this mongoose code and how might we fix it? (Hint: Assuming there is a document with a name of "Bob", why is results not an author model on the second line?)
 
-What's wrong with this mongoose code and how might we fix it?
-(Hint: Assuming there is a document with a name of "Bob", why is results not an author model on the second line?)
+ var results = AuthorModel.find({name: "Bob"});
+ console.log(results);
 
-```js
-var results = AuthorModel.find({name: "Bob"});
-console.log(results);
-```
 
-```js
-// Your answer...
-```
+// The code does not use the callback function required in the Mongoose DB. You could call it using function(results) as the second attribute to the find function.
 
-### Question #3
+ ### Question #3
 
 Convert the following ActiveRecord sequence to Mongoose:
 
-```rb
-@andy = Instructor.find_by(name: "Andy")
-@andy.wishlist_items.create(description: "Resin Laying Deer Figurine, Gold")
-```
+Instructor.findOne({name: "Andy"}, function(err, andy){
+  andy.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
+  andy.save();
```

Comment:
This still should have a callback function as well after `andy.save(function(err, results))`